Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -1587,6 +1587,30 @@ In order to test for proper updating: try downloading an older version of the ap
 Next, verify whether or not you can continue to use the application without updating it. If an update prompt is given, verify if you can still use the application by canceling the prompt or otherwise circumventing it through normal application usage. This includes validating whether the backend will stop calls to vulnerable backends and/or whether the vulnerable app-version itself is blocked by the backend.
 Lastly, see if you can play with the version number of a man-in-the-middled app and see how the backend responds to this (and if it is recorded at all for instance).
 
+## Testing App Notifications
```

Comment:
I just realised we need to give it a reference to a MSTG-ID. 

I think there are two options:

- MSTG-STORAGE-7: No sensitive data, such as passwords or pins, is exposed through the user interface.
- MSTG-STORAGE-4: No sensitive data is shared with third parties unless it is a necessary part of the architecture.

@cpholguera I would tent to reference both, as there are two attack vectors. First about exposing the data in the notification in the UI (MSTG-Storage-4) and exposing the same information to other apps that are able to read it (MSTG-Storage-7).